packages/nx/src/tasks-runner/run-command.ts

```diff
@@ -499,21 +493,10 @@ export function getRunnerOptions(
     ...nxArgs,
   };
 
-  // NOTE: we don't pull from env here because the cloud package
-  // supports it within nx-cloud's implementation. We could
-  // normalize it here, and that may make more sense, but
-  // leaving it as is for now.
-  if (nxJson.nxCloudAccessToken && isCloudDefault) {
```

Comment:
This seems unrelated? should this actually have changed?

With these changes, this would...  add this info to all runners, even non-nx-cloud runners, if it's in `nx.json`